Zeplin.sketchplugin/Contents/Sketch/export.cocoascript

```diff
@@ -164,3 +164,13 @@ var onRun = function (context) {
     applicationPath = nil;
     path = nil;
 }
+
+var shortcutHelp = function (context) {
+    var alert = [NSAlert alertWithMessageText:@"Shortcut changed to ⌃⌘E" defaultButton:@"Continue and Export" alternateButton:@"Cancel" otherButton:nil informativeTextWithFormat:@"Due to recent shortcuts tweaks on Sketch, Zeplin's “Export Selected Artboards…” shortcut is now ⌃⌘E.\n\n☝️ That's Command, Control, E."];
```

Comment:
After seeing the `That's Command, Control, E.` part live, I feel like it needs something to highlight the shortcut, something like `That's "Command, Control, E."` or `That's `Command, Control, E.`"

Reply:
Nevermind, now that I think about it again, we can just leave this as it is.